core/root.sol

```diff
@@ -6,30 +6,42 @@ import { Dmap } from './dmap.sol';
 
 contract RootZone {
     Dmap    public immutable dmap;
-    uint256 public           last;
-    bytes32 public           mark;
     uint256        immutable FREQ = 31 hours;
     bytes32        immutable LOCK = bytes32(uint256(1 << 255));
+    bytes32 public           dark;
+    bytes32 public           mark;
+    uint256 public           term;
+    address public           user;
 
+    event Ante(uint256 indexed pile);
     event Hark(bytes32 indexed mark);
     event Etch(bytes32 indexed name, address indexed zone);
 
-    error ErrPending();
     error ErrExpired();
     error ErrPayment();
+    error ErrPending();
     error ErrReceipt();
 
     constructor(Dmap d) {
         dmap = d;
     }
 
-    function hark(bytes32 hash) external payable {
-        if (block.timestamp < last + FREQ) revert ErrPending();
-        if (msg.value != 1 ether) revert ErrPayment();
-        (bool ok, ) = block.coinbase.call{value:(10**18)}("");
+    function ante(bytes32 hash) external payable {
+        uint256 balance = address(this).balance;
```

Comment:
blocking fallback is not sufficient to block receiving eth (see 'mailman' pattern)
this quantity needs to be explicit